### DISCUSSION TITLE
“版本选择”直接转路径

### DISCUSSION THREAD

**Discussion by kong-que-dnf123** [Meloong-Git/PCL#2674] [提问] [answered]
我们服务器群发的整合包
不想用里面的PCL就直接用“添加已有文件夹”把服务器的.minecroft文件载入了
服务器更新的话我就要解压后将原来的文件夹<移出列表>然后重新<添加已有文件夹>
就导入这方面可不可以支持 
**在文件夹列表中直接修改文件夹指向路径**
![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/2c2e435d-ff24-4e86-a1ad-75d3ee7d99a1)

**Comment by Pigeon0v0** [comment]
你可以把原来的路径里面的文件替换掉.jpg

**Reply by kong-que-dnf123** [reply]
有道理
但是吧
路径查不到.minecroft的话就会**识别失败**啊
他好像不会查路径的**子文件**uwu

**Comment by zkitefly** [comment, marked as answer]
去提 issue 吧

**Reply by kong-que-dnf123** [reply]
也行awa

**Reply by kong-que-dnf123** [reply]
（才发现新功能提案申请就在issue里...）
